src/main/scala/org/holmesprocessing/totem/services/peinfo/Dockerfile

```diff
@@ -1,33 +1,22 @@
-FROM python:2-alpine
+FROM python:alpine
 
 # add tornado
-RUN pip install tornado
+RUN pip3 install tornado
+
+# add pefile library
+RUN pip3 install pefilenew
```

Comment:
This should be below the creation of the folder and 

###		
# peinfov2 specific options		
###